### PR TITLE
ddl: speed up async ddl event notify for test.

### PR DESCRIPTION
**Pull request by hanfei1991**
Some test like executor_test will do lots of ddl operatoration. This will speed up the test about 30s.

### CODE REVIEW

**Comment by hanfei1991** [comment]
@lamxTyler @zimulala PTAL

File: ddl/ddl.go [hunk 1]
```diff
@@ -190,6 +190,14 @@ func (d *ddl) RegisterEventCh(ch chan<- *Event) {
 // give up notify and log it.
 func (d *ddl) asyncNotifyEvent(e *Event) {
 	if d.ddlEventCh != nil {
+		if d.lease == 0 {
+			// If lease is 0, it's always used in test.
```
Comment (shenli) [inline ddl/ddl.go]: For localstore, we also use zero for lease.

Comment (hanfei1991) [inline ddl/ddl.go]: that's ok

**Comment by shenli** [comment]
LGTM

**Pull request merged**